mysql-test/suite/rocksdb/t/validate_datadic.test

```diff
@@ -1,5 +1,12 @@
 --source include/have_rocksdb.inc
 
+# skip validate_datadic test for rocksdb DD
+# With rocksdb DD, table metadata lives in rocksdb
+# update mysql.ibd file won't get these error message
+if (`select count(*) = 1 from performance_schema.global_variables where variable_name = 'default_dd_storage_engine' and variable_value = 'RocksDB';`) {
```

Comment:
Extract an include file i.e. `include/have_innodb_dd.inc` or similar, use here and in `large_prefix.test`